deploy/spoke/clusterrole.yaml

```diff
@@ -0,0 +1,36 @@
+# Clusterrole for work agent in addition to admin clusterrole.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: open-cluster-management:work:agent
+rules:
+# Allow agent to get/list/watch/create/delete crds.
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "create", "delete", "update"]
+# Allow agent to create/delete namespaces, get/list are contained in admin role already
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create", "delete"]
```

Comment:
It's unusual to have mutation, but not get privileges.  Intentional?

Reply:
As the comments above the config says that get/list are contained in admin role already, which will be bound to the same service account.